media-video/mpv/mpv-9999.ebuild

```diff
@@ -216,7 +208,6 @@ src_configure() {
 		$(meson_feature libmpv plain-gl)
 		$(mpv_feature_multi opengl X gl-x11)
 		$(mpv_feature_multi opengl aqua gl-cocoa)
-		$(meson_feature raspberry-pi rpi)
```

Comment:
I merged the changes for 9999 given this happened: https://github.com/mpv-player/mpv/commit/343a5fd345a84572ea5693a805119067d5b0fbc8